src/main/kotlin/com/beust/kobalt/plugin/publish/BintrayApi.kt

```diff
@@ -62,7 +62,7 @@ class BintrayApi @Inject constructor(val http: Http,
                            @Path("publish") publish: Int,
                            @Body file: File): Call<BintrayResponse>
 
-        class UpdateVersion(val desc: String?, val vcsTag: String?)
+        class UpdateVersion(val desc: String?, val vcs_tag: String?)
```

Comment:
I'd rather use the Gson annotation than renaming the Kotlin field (what is is, `@Field(name="")` or something like that?